poly/pvss.go

```diff
@@ -0,0 +1,298 @@
+package poly
+
+import (
+	"errors"
+
+	"github.com/dedis/cothority/crypto"
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/random"
+)
+
+// This package implements public verifiable secret sharing (PVSS) using
+// non-interactive zero-knowledge (NIZK) proofs to show equality of discrete
+// logarithms (DLE).
```

Comment:
OK, I see there already is such documentation in this package... We could create a `proof` or `pvss` sub-package for this, but I'm not sure this a good idea.